doc/development_workflow.md

```diff
@@ -0,0 +1,60 @@
+# Development process in Status Open Bounty
+
+We have a continuously deployed version tracking the `develop` branch live at [test environment](https://openbounty.status.im:444). It uses the [Ropsten](https://ropsten.io/) Ethereum testnet. Any one is welcome to use it and contribute to Open Bounty.
+ 
+Any help is greatly appreciated!
+Currently we use two projects -  `Pipeline For Issues` and `Pipeline For Pull Requests`, issues and pull requests in our repository are passing through all or several stages described below.
+
+## Pipeline For Issues
+
+Issues are added to this project by product owner.
+
+#### To define
+This is backlog for all features/issues/enhancements which we want to include to development process. 
+
+All issues here should be marked with:
+* type - `bug`, `tech-debt`,` enhancement`, `proposal` labels
```

Comment:
What is the advantage of having `tech-debt` issues in this workflow? The only one I can think of is prioritization, but the rest of the workflow (definition and design) doesn't help much in technical debt.

Also: how about having proposals outside the workflows, and only once they evolve from proposal to wanted issue they get this workflow? Issues in `To Define` should probably be more than a proposal, and `To Define` is the first step of this workflow.

Reply:
> So, I don't see the reason to exclude these issues from Pipeline for issues

What I try to find is a reason to include it, not exclude it. :))

But totally okay with having them.

And of course the associated pull requests will have to go through `Pipeline for issues`, I just meant to the issues. :)